browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -237,11 +237,25 @@ JSDialog.combobox = function (parentContainer, data, builder) {
 
 		resetSelection();
 		for (var i in entries) {
-			if (entries[i] == this.value) {
+			if (entries[i].text == this.value) {
 				entries[i].selected = true;
 				break;
 			}
 		}
+		// check for drop down is in open state or not.
+		// If open then we should make focus in entries field for Arrow key navigation
+		if (builder.map.jsdialog.hasDropdownOpened()) {
```

Comment:
I would put `event.key === 'ArrowDown'` as a first condition, so we will not enter this block when not needed without executing unnecessary code